12/umbraco-vendr/core/how-to-guides/migrating-from-vendr-to-umbraco-commerce.md

```diff
@@ -316,4 +329,6 @@ dotnet add package Umbraco.Commerce.Core
 
 ### Step 2: Update implementations
 
-The final step in the migration is to update all async methods to accept an additional `CancellationToken` parameter.
+The final step in the migration is to update all abstract async methods exposed by the base class. It needs to be updated to accept an additional `CancellationToken cancellationToken = default` parameter as the final method argument. Your Integrated Development Environment (IDE) should provide feedback on all the methods that have been updated.
+
+[^1]: Vendr previously used the `.js` file extension to serve JSON files without having to configure a new `.json` mimetype on the server. This is no long necesarry in Umbraco Commerce so we can now use the correct `.json` file extension.
```

Comment:
Oops it should ready "This is no longer necessary" not "This is no long necesarry"